### DISCUSSION TITLE
Importing scene date from file's creation date?

### DISCUSSION THREAD

**Discussion by Rag3Quid** [stashapp/stash#3635] [Q&A] [answered]
Is there a way to (batch) import the creation date of my video files and set it to the respective scene's date?
I know you can sort by "file modification time" but the "recently released scenes" tab doesn't sort the scenes correctly if there's no scene-date set and I'd also prefer sorting by creation time rather than modification time.

**Comment by Flashy78** [comment, marked as answer]
https://docs.stashapp.cc/in-app-manual/tasks/scenefilenameparser/

**Reply by Rag3Quid** [reply]
This is for if I've got a date in the filename, right?
So I could write (or find) a script that adds the creation date to the names of all my video files, use the parser to add date to scene and then get another script that removes the date again after i'm done, if I don't want to keep the date in the filename...
Sounds relatively simple, but I'm a little afraid I'd mess my files up somehow, since I can barerly read code.
There's no way to directly grab the date directly from the video's metadata?

**Reply by Flashy78** [reply]
Ah, my bad, I thought it was in your filename.
I'm not aware that anyone has written one, but the easiest way would probably be a Stash plugin that runs by the user and simply reads the timestamp of the file and updates the scene in Stash.
https://docs.stashapp.cc/in-app-manual/plugins/
A few existing ones that could be used as a starting point https://docs.stashapp.cc/add-ons/plugins/ some are similar, but don't do exactly what you want.

**Reply by Rag3Quid** [reply]
Gotcha. Well, I guess writing a plugin is even more out of my reach than that workaround with the filename parser.. But I'll look into it. ^^
Thanks for the help.
